golang/cosmos/x/vstorage/keeper/grpc_query.go

```diff
@@ -30,6 +38,209 @@ func (k Querier) Data(c context.Context, req *types.QueryDataRequest) (*types.Qu
 	}, nil
 }
 
+// ===================================================================
+// /agoric.vstorage.Query/CapData
+// ===================================================================
+
+const (
+	// Media types.
+	JSONLines = "JSON Lines"
+
+	// CapData transformation formats.
+	FormatCapDataFlat = "flat"
+
+	// CapData remotable value formats.
+	FormatRemotableAsObject = "object"
+	FormatRemotableAsString = "string"
+)
+
+var capDataResponseMediaTypes = map[string]string{
+	JSONLines: JSONLines,
+	// Default to JSON Lines.
+	"": JSONLines,
+}
+var capDataTransformationFormats = map[string]string{
+	FormatCapDataFlat: FormatCapDataFlat,
+	// Default to no transformation.
+	"": "",
+}
+var capDataRemotableValueFormats = map[string]string{
+	FormatRemotableAsObject: FormatRemotableAsObject,
+	FormatRemotableAsString: FormatRemotableAsString,
+	// No default.
```

Comment:
with no default, is it an error when the request omits this parameter? `object` seems to me the appropriate default as it's the most generic.

Reply:
Yes, I decided that omitting FormatRemotableAsString should be an error because all valid values are lossy w.r.t. CapData.

Reply:
ah I see. that rationale would be worth including in the comment

Reply:
Done.